components/ChooseOpponent.js

```diff
@@ -7,133 +7,132 @@ import CarolineEllisonImage from "../public/images/Caroline_Ellison.jpg";
 import SBFImage from "../public/images/Sam_Bankman_Fried.jpg";
 
 const ChooseOpponent = () => {
-  const betAmount = useBettingStore((state) => state.betAmount);
   const opponent = useBettingStore((state) => state.bonkAgainst);
   const pickOpponent = useBettingStore((state) => state.setBonkAgainst);
 
   return (
-    <div className="container py-5 px-6 mx-auto">
-      <section className="mb-10 text-black-400 text-center">
-        <h1 className="text-3xl font-bold mb-12">
-          Choose who to <div className="text-orange-400">BONK</div> for{" "}
-          {betAmount} against {opponent}
-        </h1>
+    <div>
+      <section>
+        <div className="mb-12"></div>
+        <div class="py-6 sm:py-8 lg:py-12">
+          <div class="max-w-screen-2xl px-4 md:px-8 mx-auto">
+            {/* <!-- text - start --> */}
+            <div class="mb-10 md:mb-16">
+              <h1 class="text-gray-800 text-2xl lg:text-4xl font-bold text-left mb-4 md:mb-6">
+                BONK <h2 class="text-gray-100">{opponent}</h2>
```

Comment:
Add the `opacity-0` and `select-none` classes to the h2 element and change `opponent` to `opponent || <placeholder>` in order to prevent the h2 element from pushing the content beneath it down on the page once an opponent is selected.